dev-support/ranger-docker/README.md

```diff
@@ -81,9 +81,9 @@ docker-compose -f docker-compose.ranger-base-ubi.yml build --no-cache
 # To enable file based sync source for usersync do:
 # export ENABLE_FILE_SYNC_SOURCE=true
 
-# valid values for RANGER_DB_TYPE: mysql/postgres/oracle
+# valid values for RANGER_DB_TYPE: mysql/postgres/oracle/sqlserver
 
-docker-compose -f docker-compose.ranger.yml -f docker-compose.ranger-${RANGER_DB_TYPE}.yml -f docker-compose.ranger-usersync.yml -f docker-compose.ranger-tagsync.yml up -d
+docker-compose -f docker-compose.ranger.yml -f docker-compose.ranger-usersync.yml -f docker-compose.ranger-tagsync.yml up -d
```

Comment:
Other references to `docker codocker-compose.ranger-${RANGER_DB_TYPE}.yml` in README.md should be removed as well.